### PR TITLE
Feature/1794 security audit

### PR DESCRIPTION
**Pull request by timohuber**

### CODE REVIEW

File: pool/app/pool_user/entity.ml [hunk 1]
```diff
@@ -17,27 +17,113 @@ module PasswordConfirmed = struct
...
+           false
+      |> function
+      | true -> Ok p
+      | false -> Error PoolError.PasswordPolicyCapitalLetter
+    ;;
+
+    let validate_number p =
+      p
+      |> CCString.to_list
+      |> CCList.fold_left
```
Comment (mabiede) [inline pool/app/pool_user/entity.ml:50]: same function as above with different boundaries, extract common part

File: pool/database/seed/seed_user.ml [hunk 2]
```diff
@@ -243,7 +243,7 @@ let contacts db_label =
   in
   let password =
     Sys.getenv_opt "POOL_USER_DEFAULT_PASSWORD"
-    |> CCOption.value ~default:"user"
+    |> CCOption.value ~default:"Password1!"
```
Comment (mabiede) [inline pool/database/seed/seed_user.ml:246]: The `admin` password isn't changed, on purpose?

File: pool/test/tenant_test.ml [hunk 3]
```diff
@@ -81,7 +81,7 @@ module Data = struct
     let server = "smtp.uzh.ch"
     let port = 587
     let username = "engineering@econ.uzh.ch"
-    let password = "emailemail"
+    let password = "Password!"
```
Comment (mabiede) [inline pool/test/tenant_test.ml:84]: Number is missing

**Review by mabiede — CHANGES_REQUESTED** [review]

File: pool/database/seed/seed_user.ml (hunk 2 above)
Comment (timohuber) [reply pool/database/seed/seed_user.ml:246]: Wasn't necessary as we use the Sihl `create_admin` function to seed admins. But I updated this pw as well.

File: pool/test/tenant_test.ml (hunk 3 above)
Comment (timohuber) [reply pool/test/tenant_test.ml]: 😄 That is the smtp password. I resented it to the old one.

I don't think we need a password policy for smtp passwords. That should be part of the email service, shouldn't it?

File: pool/app/message_template/default/locales/default_de.ml [hunk 4]
```diff
@@ -21,6 +21,45 @@ let add_salutation_to_text =
 
 let add_salutation html = div ((salutation :: html) @ [ complimentary_close ])
 
+let account_suspension_notification =
+  let label = Label.AccountSuspensionNotification in
+  let email_text =
+    [ p
+        [ txt
+            "aufgrund zu vieler fehlgeschlagener Anmeldeversuche wurde Ihr \
```
Comment (mabiede) [inline pool/app/message_template/default/locales/default_de.ml:29]:
```suggestion
            "Aufgrund zu vieler fehlgeschlagener Anmeldeversuche wurde Ihr \
```

File: pool/app/message_template/default/locales/default_en.ml [hunk 5]
```diff
@@ -13,6 +13,45 @@ let add_salutation_to_text =
 
 let add_salutation html = div ((salutation :: html) @ [ complimentary_close ])
 
+let account_suspension_notification =
+  let label = Label.AccountSuspensionNotification in
+  let email_text =
+    [ p
+        [ txt
+            "due to too many failed login attempts, your account has been \
```
Comment (mabiede) [inline pool/app/message_template/default/locales/default_en.ml:21]:
```suggestion
            "Due to too many failed login attempts, your account has been \
```

**Review by mabiede — APPROVED** [review]

**Pull request merged**